Next/src/mock/index.ts

```diff
@@ -45,3 +46,28 @@ export const data: Element[] = [
     ],
   },
 ];
+
+export const blogdata: BlogElement[] = [
+  {
+    type: "image",
+    url: "https://media.csesoc.org.au/content/images/size/w2000/2022/03/studyTips-01-1.png",
+  },
+  {
+    type: "title",
+    children: [
+      {text: "ARTICLES"}
+    ]
+  },
+  {
+    type: "headline",
+    children: [
+      {text: "Optimise Your Study Life:Study Apps", bold: true}
+    ]
+  },
+  {
+    type: "paragraph",
+    children: [
+      {text: "In the second part of the series, Alex Xu walks us through study apps you can use to boost your productivity!"}
+    ]
+  }
+]
```

Comment:
is this the most efficient way to present the data? 

Firstly, having an array represent sections of 1 container doesnt make that much sense since at a top level, 1 object should represent 1 container, and all sections should be nested inside the container object as sections

Secondly, is it necessary to have the children tag representing the content of an object
For example:
```
{
  type: "title",
  text: "articles"
}
```
is sufficient, and similarly with everything else, 
there is no need to create more un-necessary nesting

Thirdly, I dont think for your container, it is not necessary to introduce section block mapping, i.e. utilising the `map` function to map out individual sections.